packages/embark/src/lib/utils/template_generator.js

```diff
@@ -119,10 +119,9 @@ class TemplateGenerator {
 
     if (installPackages) {
       console.log(__('Installing packages...').green);
-      utils.runCmd('npm install', null, (err) => {
+      utils.runCmd('npm install', {exitOnError: false}, (err) => {
         if (err) {
-          console.error(utils.errorMessage(err).red);
-          process.exit(1);
+          console.error(__('Could not install dependencies. Try running `npm install` inside the project directory.').red);
```

Comment:
Just a tiny nitpick: We might wanna reword this to not explicitly asking for `npm install`. Maybe we'll have a case in the future were a template actually in fact requires `yarn` ?

Just thinking out loud here, but not a big deal.